python/cudf/cudf/core/udf/groupby_typing.py

```diff
@@ -206,11 +300,30 @@ def generic(self, args, kws):
         return nb_signature(self.this.index_type, recvr=self.this)
 
 
-class GroupCorr(AbstractTemplate):
+class GroupCorr(GroupBinaryAttrBase):
     key = "GroupType.corr"
 
-    def generic(self, args, kws):
-        return nb_signature(types.float64, args[0], recvr=self.this)
+
+def _make_unsupported_dataframe_attr(attrname):
+    def _attr(self, mod):
+        raise UDFError(
+            f"JIT GroupBy.apply() does not support DataFrame.{attrname}(). "
+            f"To see what's available, visit {_UDF_DOC_URL}"
+        )
+
+    return _attr
```

Comment:
I don't see this function used anywhere.

Reply:
Removed!